libs/designsystem/form-field/src/input/input.component.stories.ts

```diff
@@ -30,7 +30,7 @@ export const Input: Story = {
     inputmode: '',
     placeholder: '',
   },
-  render: (args: InputComponent) => ({
+  render: (args) => ({
```

Comment:
<img width="479" alt="image" src="https://github.com/user-attachments/assets/061964fb-362b-4b83-90ce-3341f75cf738">
These are the peerDeps of the @storybook/angular version 7.6.13 that we currently use - and it looks like it only supports up to angular version 18 (not including that version) - so I believe we have to upgrade storybook

Reply:
I see, then I think we should do the storybook upgrade in a separate PR before we merge this one, and remove the args type annotation throughout 👍 

Then we can focus on storybook first, and fix any issues. For instance, I can see that `VisualTestingReadme.mdx` is currently failing to load inside storybook because of the upgrade.